docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -0,0 +1,268 @@
+   ```yaml
+   scrape_configs:
+     - job_name: myapp
+       scrape_classic_histograms: true
+   ```
+
+   in your scrape jobs, to get both histogram version.
+
+   {{% admonition type="note" %}}
+   Native histograms don't have a textual presentation at the moment on the application's `/metrics` endpoint, thus Prometheus negotiates a Protobuf protocol transfer in this case.
+   {{% /admonition %}}
+
+1. To be able to send native histograms to a Prometheus remote write compatible receiver, for example Grafana Cloud Metrics, Mimir, etc, set `send_native_histograms` to `true` in the remote write configuration, for example:
+
+   ```yaml
+   remote_write:
+     - url: http://.../api/prom/push
+       send_native_histograms: true
+   ```
+
+### Scrape and send native histograms with Grafana Agent
+
+Use the latest version of the Grafana Agent in [Flow mode](/docs/agent/latest/flow/) or at least version 0.37.
+
+1. To enable scraping native histograms you need to enable the argument `enable_protobuf_negotiation` in the `prometheus.scrape` component:
+
+   ```
+   prometheus.scrape "myapp" {
+     enable_protobuf_negotiation = true
+   }
+   ```
+
+1. The above flag will make Grafana Agent detect and scrape native histograms, but ignores classic histogram version of metrics. In case of a migration when a metric is both a classic and native histogram at the same time you need to set `scrape_classic_histograms` to `true` in your scrape jobs, for example:
+
+   ```
+   prometheus.scrape "myapp" {
+     enable_protobuf_negotiation = true
+     scrape_classic_histograms = true
+   }
+   ```
+
+   {{% admonition type="note" %}}
+   Native histograms don't have a textual presentation at the moment on the application's `/metrics` endpoint, thus Grafana Agent negotiates a Protobuf protocol transfer in this case.
+   {{% /admonition %}}
+
+1. To be able to send native histograms to a Prometheus remote write compatible receiver, for example Grafana Cloud Metrics, Mimir, etc, set `send_native_histograms` argument to `true` in the `prometheus.remote_write` component, for example:
+
+   ```
+   prometheus.remote_write "mimir" {
+     endpoint {
+       url = "http://.../api/prom/push"
+       send_native_histograms = true
+     }
+   }
+   ```
+
+## Migrate from classic histograms
+
+It is perfectly possible to keep the custom bucket definition of a classic histogram and add native histogram buckets at the same time. This can ease the migration process, which can look like this in general:
+
+1. Add native histogram definition to an existing histogram in the instrumentation.
+1. Let Prometheus or Grafana Agent scrape both classic and native histogram.
+1. Send native histograms to remote write - if classic histogram is scraped, it is sent by default.
+1. Start modifying the recording rules, alerts, dashboards to use the new native histograms.
+1. Once everything works, remove the custom bucket definition (`Buckets`/`classicUpperBounds`) from the instrumentation.
```

Comment:
Or just remove the `scrape_classic_histograms: true` from the scrape config?

Reply:
I'll add as a final step when you got through all histograms. Because an advantage of this approach is that you can be iterative.

Reply:
No, you're right. I'll add as "or" , flows more natural.